cekit-helpers/cekit-cache-server/src/main/java/org/kie/cekit/cacher/builds/nightly/NightlyBuildsPullRequestAcceptor.java

```diff
@@ -157,12 +136,12 @@ public void onFilePersisted(String fileName, String checkSum) {
 
                                 log.fine(String.format("Updating Business Central rhpam_business_central_distribution.zip from [%s] to [%s]", artifact.getMd5(), bcCheckSum));
                                 artifact.setMd5(bcCheckSum);
-                                yamlFilesHelper.writeModule(businessCentral, businessCentralFile);
+                                yamlFilesHelper.writeModule( buildUtils.businessCentral(),  buildUtils.businessCentralFile());
```

Comment:
removing white spaces, please apply this change in all needed places
```suggestion
                                yamlFilesHelper.writeModule(buildUtils.businessCentral(), buildUtils.businessCentralFile());
```